test/test_exception.py

```diff
@@ -133,6 +141,9 @@ def test_unknown_error(self):
         self.assertTrue(isinstance(error, UnknownError))
         self.assertEqual('Unknown error: 499 code message', str(error))
 
-    def _check_one(self, expected_class, status, code, message, post_params=None):
-        actual_exception = interpret_b2_error(status, code, message, post_params)
+    def _check_one(
+            self, expected_class, status, code, message, post_params=None, response_headers=None
```

Comment:
yapf still flags this line (and rightly so). For such long function signatures, put a `,` after last argument and yapf will make it multi-line automatically.